pkg/skaffold/deploy/helm.go

```diff
@@ -51,7 +50,7 @@ import (
 
 var (
 	// versionRegex extracts version from "helm version --client", for instance: "2.14.0-rc.2"
-	versionRegex = regexp.MustCompile(`\"v(\d[\w\.\-\.]+)`)
+	versionRegex = regexp.MustCompile(`v(\d[\w.\-]+)`)
```

Comment:
was this regex mismatching before? I don't doubt your change here, just want to understand.

Reply:
I think an Image is worth more than a thousands words.
![dummy](https://user-images.githubusercontent.com/13263011/80796951-d4d55380-8ba0-11ea-8077-7f5bedb58397.png)

LSS my IDE recommended. I can roll it back if you want

Reply:
These seem fine to me, they are redundant - the `"` is not really required as `v` identifies the start of the string (although you could argue that potentially strings starting with v but not immediately after a `"` will be matched too - shouldn't matter in this output though).  Also, in this case. as we are in a character set (`[...]`), the two `\.`s are superfluous and in a character set you don't need to escape `.`.